### DISCUSSION TITLE
Hbstacks and Obsidian.md

### DISCUSSION THREAD

**Discussion by luigi8bits** [hbstack/.github#92] [Q&A] [answered]
Hi, I have been using hbstacks to create my public "digital garden." Now, I am migrating my notes to an [Obsidian.md](https://obsidian.md) vault. 

My question is: is there a way to publish these notes from Obsidian to hbstacks, similar to "[quartz](https://github.com/jackyzha0/quartz)" and "[digital-garden](https://github.com/oleeskild/obsidian-digital-garden)"?

I’m not an expert in development stuff, but I know that Hugo uses Markdown files, and so does Obsidian. @razonyang , do you have any tips or ideas on whether this is possible?

**Comment by razonyang** [comment, marked as answer]
Hi, if I understand correctly, you want to use Obsidian as the editor for the content, and publish the content/notes with Hugo and HB themes.

If so, it's feasible, to do so:

1. Create Obsidian vault with **Open folder as vault**.
2. And then **Open** the path to your Hugo project.
![image](https://github.com/user-attachments/assets/933f8d11-a36d-458c-947b-47835ae4dbb6)
3. Once done, navigate to the vault's setting panel.
4. Modify **Files and links**, such as the notes folder, excluding files, etc.
![image](https://github.com/user-attachments/assets/a68e9dec-6900-48c9-b47a-2f74803948d4)
5. Enable the **Properties** plugin.
![image](https://github.com/user-attachments/assets/e5663033-1f79-46f7-b11d-849ef5c3ffc7)
6. The preview and deployment process is the same as Hugo.
7. There are **no changes required** in the existing Hugo project, as Obsidian is an editor in this case.

To be honest, I don't have much experience with Obsidian. I think there are some limitations about it, such as the `[[Wikilinks]]` which is not supported by Hugo/HB now.

Please feel free to create an issue for a bug or a feature request on here.

**Reply by luigi8bits** [reply]
Wowwww, very fast answer! ❤️ Yes, this is exactly the kind of stuff I like to do. Thanks! I’ll try it out and let you know.

**Reply by razonyang** [reply]
No problem.

Although [Hugo doesn't support wikilink](https://github.com/gohugoio/hugo/issues/3606#issuecomment-1555955974), there is a performance trade-off workaround (replace string with regexp) that transforms wikilinks into Hugo shortcodes.

Please let me know if you would like to use wikilinks.
